CONTRIBUTING.md

```diff
@@ -185,6 +190,31 @@ If you are having trouble with links to sections or code blocks, it might
 be due to duplicated sections, or to spaces or other characters. To
 preview the generated links, use the `myst-anchors` tool.
 
+### Troubleshooting
+
+General advice is to use the `sphinx-build` command directly instead of `make`:
+`sphinx-build -vvv -b html src build/`. The `-vvv` flag should increase the
+verbosity of logs and give you more to work with to troubleshoot your issue.
+
+It is also worth removing build directories and files such as `_build`, `build`,
+and [`~/.cache/salad`](https://github.com/common-workflow-language/user_guide/issues/268#issuecomment-1276068865).
+You may want to start afresh with a new Python virtual environment,
+installing the dependencies from scratch.
+
+If running `make html` or `make watch` appears to run fine but [gets stuck
+](https://github.com/common-workflow-language/user_guide/issues/268) at some
+part of the process (`10%`, `33%`, `57%`, etc.) for longer than 2-5 minutes
+you may have to try the following:
+
+1. Check if you have the latest versions installed in your Python virtual environment.
```

Comment:
Latest versions of what?

Reply:
Ops. Dependencies. Let me update after adding verbosity to make. Thanks!